spec/mailers/previews/user_mailer_preview.rb

```diff
@@ -1,4 +1,4 @@
-# Preview all emails at http://localhost:3001/rails/mailers/user_mailer
+# Preview all emails at http://localhost:3000/rails/mailers/user_mailer
```

Comment:
Again, I am just undoing a change I shouldn't have made to the original Klaxon work!